com.unity.render-pipelines.core/Runtime/Textures/RTHandles.cs

```diff
@@ -90,6 +90,52 @@ public static class RTHandles
             );
         }
 
+        /// <summary>
+        /// Allocate a new fixed sized RTHandle with the default RTHandle System.
+        /// </summary>
+        /// <param name="descriptor">RenderTexture descriptor of the RTHandle.</param>
+        /// <param name="slices">Number of slices of the RTHandle.</param>
```

Comment:
The slices parameter does not exist (anymore?).

Reply:
Also msaaSamples and other overloads have unexisting parameters documented.

Reply:
Good catch! Those two params have now been removed.